### ISSUE TITLE
[Seed-vc-v2] Training script of the VQ model in v2

### ISSUE THREAD

**Issue by josephwong14wkh** [Plachtaa/seed-vc#193]
Hi, thank you for the great work. 

I saw into the code and realized that we are not able to train / finetune the VQ in the model. As i want to finetune the model on a new language (i.e. Cantonese in my case), may i know how to fine tune the VQ components? Will you plan to release the full training script of the model?

**Comment by Plachtaa** [comment]
We are still working on formalizing the training code of VQ part, and will release it soon :)
